docs/helm/helm-chart-verification-via-console.md

```diff
@@ -0,0 +1,24 @@
+# Helm Chart Verification
+
+Chart Verifier is a cli tool that runs a series of independent checks defined in different profiles. This CLI tool is used on our Chart Certification Flow.
+
+The Helm Chart Verifier api `/api/helm/verify` is an integration of this cli tool  on `Openshift Developer Console`(ODC) and this will provide user more insight on whether the helm charts they are targeting will run well on OpenShift. The following checks will be run when user is looking to verify their helm chart from console:
+
+1. ContainsTest: Checks chart test files exist
+2. ContainsValuesSchema: Checks whether values schema file exists
+3. ContainsValues: Checks if values file exist
+4. HasKubeVersion: Checks kubernetes version specified
+5. HasReadme: Checks chart has a README
+6. HelmLint: checks Helm lint for the chart
+7. ImagesAreCertified: Checks if image is Red Hat certified 
+8. IsHelmV3: Checks the apiversion for the chart and whether it is used in Helm 3
+9. NotContainCsiObjects: Checks whether CSI objects are present
+10. NotContainsCRDs: Checks chart does not contain CRDs
+11. RequiredAnnotationsPresent: Checks all required annotations are present
+
```

Comment:
Suggestion: Put in table format with headers as Checks and Description (2 columns)

@Kartikey-star We Will sit together to review this tomorrow when you are available.

Reply:
@Kartikey-star Please follow the descriptions written here: https://github.com/openshift/console/pull/11964/files#r962695319